client/src/components/proof-heatmap.tsx

```diff
@@ -21,52 +20,69 @@ const ProofHeatMap = ({ roots }: { roots: Roots[] }) => {
       ? new Date(root.lastFaultedAt)
       : null
 
-    // Root was challenged and faulted in the last 7 days
-    if (lastFaultedDate && lastFaultedDate >= sevenDaysAgo) {
-      // If it was also proven successfully after the fault, it's successful
-      if (lastProvenDate && lastProvenDate > lastFaultedDate) {
-        return 'success'
-      }
-      return 'fault'
+    const recentActivity =
+      (lastProvenDate && lastProvenDate >= sevenDaysAgo) ||
```

Comment:
Where did 7 days come from?  Is it a performance requirement?  It would be nice to go back at least 30 days for the production setup since we only get one prove message per day.  But if it breaks things not worth changing